### PR TITLE
PDS standard - Github actions

### PR DESCRIPTION
**Pull request by niden**
- Added the PDS standard for the project
- Downloaded `ocular.phar` to the `bin` directory to speed things up
- Added 7.4 standard to travis
- Added php codesniffer in composer (dev)
- Enabled github actions to run the tests (also)
- Added badges to the README

Additional PHPCS PR to follow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Aura.Di
 
 [![PDS Skeleton](https://img.shields.io/badge/pds-skeleton-blue.svg?style=flat-square)](https://github.com/php-pds/skeleton)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/niden/Aura.Di/badges/quality-score.png?b=pds-standard)](https://scrutinizer-ci.com/g/auraphp/Aura.Di/?branch=pds-standard)
```
Comment (harikt) [inline README.md]: Badges are already in Quality section. https://github.com/auraphp/Aura.Di#quality . But I see the PDS skeleton badge is not there. May be we need to fix the badge in Quality section ?

**Comment by harikt** [comment]
Hi @niden ,

I am not in favor of certain things.

1. PDS skelton doesn't enforce you to have empty directories ( May be we need to confirm from @pmjones  ).
2. I don't think we need to keep ocular in the bin directory just to use for travis.
3. I don't know more about the github actions. May be you can tell the benefits it offers ?

These are some of my thoughts.

File: README.md (hunk 1 above)
Comment (niden) [reply README.md]: Ah I did not notice that sorry. I will move them around

**Comment by niden** [comment]
Hey @harikt 

Not sure about #1. I always kept directories even empty ones.
#2 I thought I would save a download there. I can move it out easily and adjust the travis/github actions
#3 This is a _new_ thing that GitHub introduced which allows you to have a full CI based on _actions_ which can pretty much do anything you need. The actions can be docker based, or javascript/typescript based etc. For instance in one of my projects I am running all the integration tess in mysql and the next run is in sqlite etc. Very flexible and I thought it might help as an addon instead or alongside Travis. 

Have a look at the actions on my fork: https://github.com/niden/Aura.Di/actions As you will see it is pretty much Travis on GitHub. I can remove it if you like - I started using it more and more so thought that Aura could benefit with one more CI.

**Comment by harikt** [comment]
Hi @niden ,

No actions need to be taken. Let other members also get a change to look into this.

May be @frederikbosch will respond once he get some time.

File: README.md (hunk 1 above)
Comment (niden) [reply README.md]: Resolved

**Comment by frederikbosch** [comment]
@harikt What shall we do with this one? Personally, I rather include PHPStan than the PDS standard.

**Comment by harikt** [comment]
@frederikbosch both ( PDS and PHPstan) are different tools I believe. I have no objections with what you choose. So full support for your choices :) .

**Comment by harikt** [comment]
What are these `public` and `resources` files to do with. I wonder whether those are compulsory for PDS. If so I don't like keeping such ones just for the sake. I am against those.

**Comment by frederikbosch** [comment]
Sorry, I rather leave things as they are now. More tools means more maintenance. Since the proposed tools do not provide anything to the code, I rather leave them out.

**Pull request closed without merging**